easybuild/easyconfigs/h/HPCX/HPCX-2.10.eb

```diff
@@ -0,0 +1,78 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = 'Tarball'
+
+name = 'HPCX'
+version = '2.10'
+
+homepage = 'http://www.mellanox.com/page/products_dyn?product_family=189&mtag=hpc-x'
+description = """The Mellanox HPC-X Toolkit
+is a comprehensive MPI and SHMEM/PGAS software suite
+for high performance computing environments"""
+
+toolchain = SYSTEM
+
+# By downloading, you accept the HPC SDK Software License Agreement (https://docs.nvidia.com/hpc-sdk/eula/index.html)
+# accept_eula = True
+source_urls = ['https://content.mellanox.com/hpc/hpc-x/v%(version)s/']
+sources = ['%(namelower)s-v%(version)s-gcc-MLNX_OFED_LINUX-5-redhat8-cuda11-gdrcopy2-nccl2.11-x86_64.tbz']
```

Comment:
HPC-x seems to need CUDA and GDRCopy to work, as well as the Mellanox OFED 5 drivers. If that is the case, I would add dependencies on CUDA and GDRCopy and explain in a comment that it need compatible hardware with MLNX_OFED 5.
Moreover, this tarball targets rhel8. I see that there is another for rhel7. We should distinguish this easyconfig by adding a `versionsuffix = -redhat8`, to allow the existence of another easyconfig for rhel7.